### PR TITLE
Remove NVDA service, no longer needed as NVDA will require Windows 7 SP1 or higher

### PR DESCRIPTION
**Pull request by josephsl**
### Link to issue number:
None

### Summary of the issue:
Removes NVDA service which was needed in Windows XP, now that Windows 7 SP1 is the minimum requirement for NVDA.

### Description of how this pull request fixes the issue:
This pull request modifies setup and installer to:

1. Tells py2exe to not build the service.
2. Do not install NVDA service nor start it.
3. Service module is removed.

### Testing performed:
Successfully compiled NVDA without the service executable.

### Known issues with pull request:
None

### Change log entry:
None

### CODE REVIEW

File: source/installer.py [hunk 1]
```diff
@@ -192,10 +192,6 @@ def registerInstallation(installDir,startMenuFolder,shouldCreateDesktopShortcut,
 			_winreg.SetValueEx(k,config.CONFIG_IN_LOCAL_APPDATA_SUBKEY,None,_winreg.REG_DWORD,int(configInLocalAppData))
 	if easeOfAccess.isSupported:
```
Comment (leonardder) [inline source/installer.py]: This makes no sense any more, as this will always be True.

**Comment by josephsl** [comment]
Hi, that one and several commits that’ll modernize the API (including always making NVDA high DPI aware) is headed for a separate PR, as it is better to do one thing at a time. Thanks.

**Comment by michaelDCurran** [comment]
A reference to nvda_service.exe must be removed from sconstruct. This is causing signed builds to fail.

**Review by michaelDCurran — CHANGES_REQUESTED** [review]
nvda_service.exe must be removed from sconstruct

**Comment by michaelDCurran** [comment]
add-ons such as goldenCursor and windows7Mangnifier depend on win32api, which is no longer included as it was an implicit dependency on nvda_service.
It would be great if these add-ons could be rewritten to not use win32api, but for this next release at least, we should manually include win32api in NVDA so these add-ons do not break.
Refer to how we explicitly include the bisect module in setup.py.

**Review by michaelDCurran — APPROVED** [review]

File: source/setup.py [hunk 2]
```diff
@@ -205,8 +205,9 @@ def getRecursiveDataFiles(dest,source,excludes=()):
 			"serial.loopback_connection", "serial.rfc2217", "serial.serialcli", "serial.serialjava", "serial.serialposix", "serial.socket_connection"],
 		"packages": ["NVDAObjects","virtualBuffers","appModules","comInterfaces","brailleDisplayDrivers","synthDrivers"],
 		# #3368: bisect was implicitly included with Python 2.7.3, but isn't with 2.7.5.
-		# Explicitly include it so we don't break some add-ons.
-		"includes": ["nvdaBuiltin", "bisect"],
+		# Also, the service executable used win32api, which some add-ons use for various purposes.
+		# Explicitly include them so we don't break some add-ons.
```
Comment (derekriemer) [inline source/setup.py]: nit: them, -> it.
Also, what about win32con for a while?

**Comment by josephsl** [comment]
Hi, I need to find out if other add-ons use that module.

**Comment by Brian1Gaff** [comment]
Is this the point though? Is it not that its a good file to keep if its only 
small as a resource unless it cocks up something that will change later on 
when you move versions?

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Comment by Brian1Gaff** [comment]
What worries me a little here is that do we know if any already written app 
modules in nvda use these files?
If they do then some software, although old may suddenly stop working.

Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
----- Original Message -----

**Comment by derekriemer** [comment]
we can (and must) fix those. They're in core.

**Comment by Brian1Gaff** [comment]
I wonder if this removal was a little premature, and what is needed is a way 
to include the bits that are always needed in the  main nvda system, and 
then only remove the rest of the service, unless people have taken 
advantage of it being a service and hence need to have the items available 
in that way of course.
 Apart from breaking more ties with XP was there really any point in 
removing the service in the first place?
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal email to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
-----

**Pull request merged**